SmartModule.toml

```diff
@@ -7,6 +7,13 @@ description = "Converts CSV into JSON"
 license = "Apache-2.0"
 visibility = "public"
 
+
+[[params]]
+name = "delimiter"
+description = "the field delimiter to use when parsing csv"
+optional = true
+
 [[params]]
```

Comment:
I am not sure if this is the right way to add multiples params

Reply:
Actually, I'm not 100% sure here either.  @morenol do you have some better insight here?

Reply:
Great!